Gemfile

```diff
@@ -122,6 +122,12 @@ group :development do
   gem 'binding_of_caller'
   # Continous execution of unit tests
   gem 'guard' # https://github.com/guard/guard
+  # Benchmarking
+  # Supporting gem for Rails Panel (Google Chrome extension for Rails development)
+  gem 'meta_request' # https://github.com/dejan/rails_panel
```

Comment:
Do we really want to merge the `meta-request` and `ruby-prof` gems into `dev` or should we remove them before merging?

Reply:
The loading time is better but not optimal so when we need to benchmark later we have the gems already installed